examples/prescriptors/neat/utils.py

```diff
@@ -2,7 +2,20 @@
 
 import os
 import subprocess
+import urllib.request
 import pandas as pd
+from validation.scenario_generator import generate_scenario
+from validation.scenario_generator import get_raw_data
+
```

Comment:
add info for downloading data if necessary, and move other path constants to top of file.